src/main/java/com/github/tomakehurst/wiremock/core/WireMockConfiguration.java

```diff
@@ -597,6 +599,12 @@ public String bindAddress() {
     return bindAddress;
   }
 
+  @Override
+  public FilenameMaker getFilenameMaker() {
+    filenameMaker = new FilenameMaker();
```

Comment:
Doesn't seem like there's any way for the user to pass their own template when using this options impl.

Which also shows we're missing some tests. Please can you add acceptance test cases, both for this way of supplying options and the CLI alternative.

Reply:
You're right. Will add the tests.